Replace angular-materialize to ng-materialize

goodmind opened the pull request:
Unfortunately, but angular-materialize package is already exists in bower

goodmind committed:
Replace angular-materialize to ng-materialize

Unfortunately, but angular-materialize package is already exists in bower

webbiesdk commented:
I don't like naming it ng-[something], that just sounds like the library is from the angular team. 
We don't want people to be confused about this project and material.angularjs.org. 

The project already in bower (https://github.com/minhongrails/angular-materialize) looks like it's dead. 
We could ask him to remove his project from bower?

krescruz commented:
The project https://github.com/minhongrails/angular-materialize is a copy this project comit in
https://github.com/krescruz/angular-materialize/tree/0390b81f679c9e1d5234c77ee98f4100aef6e5d7

goodmind committed:
Update bower.json

A possible solution, or contact the author of angular-materialize bower package

goodmind commented:
Checkout my commit

webbiesdk commented:
I just send a mail to the author of angular-materialize bower package, about removing/renaming his project in bower.

iominh commented:
Sorry, I was too trigger happy with Bower. I unregistered it so the name should be yours now. Thanks for your project and work!

webbiesdk commented:
Thanks, I'll register this project as a bower package now.

webbiesdk commented:
Well, the name is definitly ours now: 
http://bower.io/search/?q=angular-materialize
